packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -10044,6 +10060,14 @@ interface IStyleSheetConfig {
   onInsertRule?: (rule: string) => void;
 }
 
+// @public (undocumented)

```

Comment:
What is this undocumented?

Reply:
Output from [API Extractor](https://github.com/Microsoft/web-build-tools/wiki/API-Extractor)

You can see the inverse of this here: https://github.com/OfficeDev/office-ui-fabric-react/pull/7238/files

Nothing I would block on